index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Please add a Readme file. Also, remember to use [Microverse Template](https://github.com/microverseinc/readme-template)